Formula/elasticsearch-full.rb

```diff
@@ -42,9 +51,13 @@ def install
 
       bin.install libexec/"bin"/f
     end
-    bin.env_script_all_files(libexec/"bin", {})
+    if OS.mac?
+      bin.env_script_all_files(libexec/"bin", {})
+    else
+      bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.8"))
```

Comment:
Let's use the bundled JDK here, then setting this environment variable won't be necessary.